rest-api/src/main/resources/index.yml

```diff
@@ -473,8 +473,8 @@ parameters:
         type: string
         enum: [test, production, both]
         default: production
-    labelFilter:
-        name: labelFilter
+    labelFilters:
+        name: labelFilters
```

Comment:
What is this? Either it shouldn't be here, or it should remain labelFilter, we need to change the related search object for this.

Reply:
Removed this, testFilter, and complianceUnder (they were all moved and/or renamed in AdherenceReportSearch).